networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc

```diff
@@ -0,0 +1,23 @@
+[id="deploying-egress-router-cni-redirection"]
+= Deploying an egress router pod in redirect mode
+include::modules/common-attributes.adoc[]
+:context: deploying-egress-router-cni-redirection
+
+toc::[]
+
+As a cluster administrator, you can deploy an egress router pod to redirect traffic to specified destination IP addresses from a reserved source IP address.
+
+The egress router implementation uses an egress router Container Network Interface (CNI) plug-in. You can use this implementation with the OVN-Kubernetes network provider or the OpenShift SDN network provider.
```

Comment:
Possibly s/an/the/

Reply:
Reviewers, I want 2 points for this bold statement--support for OVN and SDN.  Unless it's wrong or inadvisable, of course.  Please let me know.

Reply:
this implementation *should* work with any networking plugin (we're creating an additional CNI!), but only has been tested with OVN, so if you feel comfortable with just drop that part.

Reply:
Ah, important distinction.  Between the choice of _should_ and tested, tested always wins.  Thanks very much for that info.